pkg/agent/openflow/pipeline.go

```diff
@@ -1715,6 +1790,19 @@ func (f *featureNetworkPolicy) conjunctionActionFlow(conjunctionID uint32, table
 			Done()
 	}
 	var flows []binding.Flow
+	// As IGMP and multicast use a different pipeline 'Multicast', if the rule is
+	// IGMP ingress or multicast egress，conjunctionActionFlow generates the flow
+	// to mark the packet to be allowed if policyRuleConjunction ID is matched.
+	// Any matched flow will be resubmitted to next table in corresponding metric tables.
+	if isIGMP || f.isMulticastEgressRule(tableID, isIngress, isIGMP) {
```

Comment:
I thoug here you want IGMPIngress and Egress, but it is actually matching all igmp and egress.

Reply:
since IGMP egress is filtered at the beginning, IGMP here is only ingress, I can change to `f.isIGMPIngressRule(tableID, ingress, isIGMP)`

Reply:
done